src/configure/configure.ts

```diff
@@ -648,14 +655,54 @@ class Orchestrator {
             throw error;
         }
     }
+
+    private createProvisioningConfiguration(templateId: string, branch: string, pipeplineTemplateParameters: { [key: string]: string }, mode: provisioningMode): ProvisioningConfiguration {
+        return {
+          id: null,
+          branch,
+          pipelineTemplateId: templateId,
+          pipelineTemplateParameters: pipeplineTemplateParameters,
+          provisioningMode: mode,
+        } as ProvisioningConfiguration;
+      }
+
+    private async configurePipelineRemotely(): Promise<void>{
+        const provisioningConfigurer = new ProvisioningConfigurer(this.localGitRepoHelper);
+        const template = this.inputs.pipelineConfiguration.template as RemotePipelineTemplate;
+        await provisioningConfigurer.CreatePreRequisiteParams(this.inputs);
```

Comment:
you should log telemetry for currentstep so that we can know where failues are happening

Reply:
done